uber/tests/models/test_badge_funcs.py

```diff
@@ -230,10 +230,6 @@ def staff_badges(self, session):
         # This loads badges from the session, which isn't reloaded, so the result is not always what you'd expect
         return sorted(a.badge_num for a in session.query(Attendee).filter(Attendee.badge_status != c.INVALID_STATUS).filter_by(badge_type=c.STAFF_BADGE).all())
 
-    def test_invalid_parameters(self, session):
```

Comment:
No longer needed because we're not doing the weird thing with the `direction` parameter.